src/state-summary/state-card-vacuum.html

```diff
@@ -0,0 +1,205 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+<link rel='import' href='../../bower_components/paper-dropdown-menu/paper-dropdown-menu.html'>
+<link rel='import' href='../../bower_components/paper-listbox/paper-listbox.html'>
+<link rel='import' href='../../bower_components/paper-item/paper-item.html'>
+<link rel='import' href='../../bower_components/paper-icon-button/paper-icon-button.html'>
+
+<link rel="import" href="../components/entity/state-info.html">
+<link rel="import" href="../components/entity/ha-entity-toggle.html">
+
+<dom-module id="state-card-vacuum">
+  <template>
+    <style is="custom-style" include="iron-flex iron-flex-alignment"></style>
+    <style>
+      :host {
+        @apply(--paper-font-body1);
+        line-height: 1.5;
+      }
+
+      .status-subtitle {
+        color: var(--secondary-text-color);
+      }
+
+      .container-extra-controls {
+        margin-top: 1.5em;
+      }
+
+      paper-item {
+        cursor: pointer;
+      }
+    </style>
+
+    <div class='horizontal justified layout'>
+      <state-info state-obj="[[stateObj]]" in-dialog='[[inDialog]]'></state-info>
+      <ha-entity-toggle state-obj="[[stateObj]]" hass='[[hass]]'></ha-entity-toggle>
+    </div>
+
+    <div hidden$='{{inDialog}}'>
```

Comment:
ok, but then, if I understand, **there is no need for a new state card**, only the **more-info card** is needed, right? Because the default card for the vacuum is a switch, I think.

What about the filtering of state attrs with `window.hassUtil.LOGIC_STATE_ATTRIBUTES`?